index.js

```diff
@@ -0,0 +1,37 @@
+var program = require('commander');
+var requireDir = require('require-dir');
+var package = require(__dirname + '/package.json');
+
+program.version(package.version)
+  .option('-D, --debug', 'enable debugging and debug output')
+  .option('-q, --quiet', 'quiet output, except for errors')
+  .option('-v, --verbose', 'verbose output')
+  .option('-j, --json', 'JSON data output')
+  .option('--nocolors', 'no ANSI colors in JSON output');
+
+// Common command init wrapper with error reporting.
+function init (cmd) {
```

Comment:
Not yet, alas. I was trying to keep that in mind throughout, but missed this. Should add at least the style checking to travis! I was having trouble coming up with tests in general / was lazy, though.